app/src/main/java/com/example/tasky/login/presentation/LoginScreen.kt

```diff
@@ -25,17 +23,27 @@ import androidx.compose.ui.draw.clip
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.font.FontWeight
-import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.text.input.ImeAction.Companion
+import androidx.compose.ui.text.input.KeyboardType
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import com.example.tasky.R
+import com.example.tasky.login.components.ButtonWithLoader
+import com.example.tasky.login.components.TaskyTextField
 import com.example.tasky.ui.theme.Black
+import com.example.tasky.ui.theme.LightBlue
 import com.example.tasky.ui.theme.LightGray
 import com.example.tasky.ui.theme.TaskyTheme
 
 @Composable
-fun LoginScreen() {
+fun LoginScreen(
+    isValid: Boolean,
```

Comment:
What's the use of isValid?

To show if an email is valid or not? If so, it'd make more sense to name it "isValidEmail" or similar

But... What about password?

To avoid having to pass a ton of stuff, you could just pass the screen's state (which can be defined later in your viewmodel). Ex: LoginState


And to avoid having to pass a ton of callbacks (onEmailValueChanged, onLoginClick, etc), you can create an "Event" sealed class for that screen. Ex: LoginEvent